node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -4199,8 +4427,8 @@ mod tests {
         )
     }
 
-    fn make_single_node_gossip(n: u16, mode: Mode) -> (Gossip_0v1, NodeRecord) {
-        let mut debut_node = make_node_record(n, true);
+    fn make_single_node_gossip<'a>(n: u16, mode: Mode) -> (Gossip_0v1, NodeRecord) {
```

Comment:
I cannot grasp why you didn't get warnings on your machine or later in Actions from this lifetime that is never actually used. There used to be days when Rust hollered over this. I think it should be removed anyway.